jest.config.js

```diff
@@ -1,5 +1,6 @@
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
 module.exports = {
+  moduleNameMapper: { "^~/(.*)$": "<rootDir>/$1" },
```

Comment:
Sigh, `jest` is such black magic to me sometimes. What's this for?

Reply:
See above comment!

Reply:
Ha, wow. I should have put 2-and-2 together with my two comments here.